app/views/categories/index.html.erb

```diff
@@ -0,0 +1,24 @@
+<div class="container mt-4">
+    <h2>Categories</h2>
+    <div class="row">
+        <% @categories.each do |category| 
+                #if the category has no words, it won't be displayed
+                unless category.words.size == 0 %>
```

Comment:
@ivanraybuglosa Instead of providing condition to display category or not., it is better to filter `@categories` object in the controller first. More faster that way.
Then also, please fix indentation in this area.